docs/api/features/manifold.py

```diff
@@ -69,19 +64,21 @@ def dataset_example(dataset="occupancy", manifold="all", path="images/", **kwarg
         return
 
     # Create single example
-    _, ax = plt.subplots(figsize=(9, 6))
-    oz = Manifold(ax=ax, manifold=manifold, **kwargs)
 
+    _, ax = plt.subplots(figsize=(9, 6))
     if dataset == "occupancy":
         X, y = load_occupancy()
     elif dataset == "concrete":
         X, y = load_concrete()
     else:
         raise Exception("unknown dataset '{}'".format(dataset))
-
-    oz.fit_transform(X, y)
-    oz.show(outpath=path)
-
+    if(quick==True):
```

Comment:
I recommend that we need to change above line to:
```python
if quick:
```
Add a line break above this! And additional comment explaining what this if-else block is doing would be great!